src/HealthCheck.Core/CheckResult.cs

```diff
@@ -2,8 +2,9 @@
 {
     public class CheckResult
     {
-        public string Checker { get; set; }
+        public string CheckerName { get; set; }
```

Comment:
It will be called "Checker" to be compatible with [Runscope/healthcheck](https://github.com/Runscope/healthcheck).

Reply:
Do you want me to return Checker? But it is not checker... it is his only name. Why should we copy flaws of other projects?

Reply:
To the caller of the healthcheck endpoint there is no distinction between a "checker" and its name. The checker instance is an internal object and the name is the only public reference to the checker. Therefore it makes sense to just call it "checker". I agree that _internally_ "checker" is a different concept (which contains a "name" property) but that's just not important to the user.